.github/actions/next-integration-stat/src/index.ts

```diff
@@ -228,14 +334,18 @@ async function getInputs(): Promise<{
   octokit: Octokit;
   prNumber: number | undefined;
   sha: string;
+  noBaseComparison: boolean;
   shouldExpandResultMessages: boolean;
 }> {
   const token = getInput("token");
   const shouldExpandResultMessages =
     getInput("expand_result_messages") === "true";
-  const shouldDiffWithMain = getInput("diff_base") === "main";
-  if (getInput("diff_base") !== "main" && getInput("diff_base") !== "release") {
-    console.error('Invalid diff_base, must be "main" or "release"');
+  const diffBase = getInput("diff_base");
+  const shouldDiffWithMain = diffBase === "main";
+  // For the daily cron workflow, we don't compare to previous but post daily summary
```

Comment:
Could we compare to the previous day?

Reply:
Technically, but searching last day is bit wonky and also as shared in the meeting current comparision gives not much usef info due to number inconsistency, so probably will attempt it later once we have better stable test results.